<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiciosAdd</class>
 <widget class="QWidget" name="ServiciosAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	border: none;
}
#dashboard{
	background-color: #eff9fe;
}
#cards QFrame{
	background-color: white;
	border-radius: 15px;
}
#frMain{
	background-color: white;
	border-radius: 15px;
}
QLineEdit{
	background: white;
	border: 1px solid black;
	border-radius: 3px;
}
QPushButton{
	padding: 5px;
	border-radius: 5px;
	color:white;
	background: #0d6efd;
}
QPushButton:hover{
	background: #1e7fff;
}
#btnAddProduct{
	background: #198754;
}
#btnAddProduct:hover{
	background: #2A9865;
}
QComboBox{
	border: 1px solid black;
	border-radius: 3px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="main" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <widget class="QFrame" name="frMain">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QFrame" name="frameView">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>350</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QWidget" name="widget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QFrame" name="frame">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <widget class="QLabel" name="lblClientes">
                    <property name="text">
                     <string>Cliente</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="cbClientes"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="lblMarca">
                    <property name="text">
                     <string>Detalles del vehiculo</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="leVehiculo"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="lblDescripcion">
                    <property name="text">
                     <string>Descripción del Servicio</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="leDescripcionServicio"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="lblPrecio">
                    <property name="text">
                     <string>Precio inicial del servio</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lePrecio"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item alignment="Qt::AlignTop">
                <widget class="QFrame" name="frame_5">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <item>
                   <widget class="QLabel" name="lblProductos">
                    <property name="text">
                     <string>Producto</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="cbProductos"/>
                  </item>
                  <item>
                   <widget class="QLabel" name="lblCantidad">
                    <property name="text">
                     <string>Cantidad</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="leCantidad"/>
                  </item>
                  <item alignment="Qt::AlignRight">
                   <widget class="QPushButton" name="btnAddProduct">
                    <property name="maximumSize">
                     <size>
                      <width>200</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="text">
                     <string>Añadir Producto</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frameAdd">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QFrame" name="frame_3">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <item>
                   <widget class="QTableWidget" name="productosTable"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_4">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_8">
                  <item>
                   <widget class="QPushButton" name="btnGuardar">
                    <property name="text">
                     <string>Guardar</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../PySide-Projects/intrefaceIA/resources.qrc"/>
  <include location="../../AICountDrive/interfaces/resources.qrc"/>
 </resources>
 <connections/>
</ui>
